doc/Architecture/spec-al_x-road_audit_log_events.md

```diff
@@ -0,0 +1,465 @@
+| Edit provider of management services                           | <ul><li>serviceProviderIdentifier - the new service provider identifier of the management service</li><li>serviceProviderName - the new service provider name of the management service</li></ul>                                                                                                                                                         |
+| Add member class                                               | <ul><li>code - the code of the added member class</li><li>description - the description of the added member class</li></ul>                                                                                                                                                                                                                               |
+| Edit member class description                                  | <ul><li>code - the code of the edited member class</li><li>description - the new description of the edited member class</li></ul>                                                                                                                                                                                                                         |
+| Delete member class                                            | <ul><li>code - the code of the deleted member class</li></ul>                                                                                                                                                                                                                                                                                             |
+
+#### 2.1.11 Backup and Restore Events
+
+The audit log events related to backup and restore.
+
+| Event                 | Data fields                                                                                  |
+|-----------------------|----------------------------------------------------------------------------------------------|
+| Back up configuration | <ul><li>backupFileName - the name of the created backup file</li></ul>                       |
+| Upload backup file    | <ul><li>backupFileName - the name of the uploaded backup file</li></ul>                      |
+| Delete backup file    | <ul><li>backupFileName - the name of the deleted backup file</li></ul>                       |
+| Restore configuration | <ul><li>backupFileName - the name of the backup file used to restore configuration</li></ul> |
+
+### 2.2 Security Server
+
+#### 2.2.1 Common Events
+
+The audit log events related to the UI logging.
+
+| Event        | Data fields |
```

Comment:
Is the "Data fields" column empty on purpose?

Reply:
Yes, it's the same way in the original docx.